bin/server.py

```diff
@@ -0,0 +1,80 @@
+"""Main module cusdeb-helpik that stores handlers"""
+import re
```

Comment:
You need to fix the order of import according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports).